tests/ballerina-test-integration/src/test/resources/auth/ballerina.conf

```diff
@@ -0,0 +1,25 @@
+# username->userid mapping
+[isuru]
+userid="user1"
+groups="prq,lmn"
+[ishara]
+userid="user2"
+groups="xyz"
+[application]
+userid="user3"
+
+# user data
+[user1]
+password="CD2EB0837C9B4C962C22D2FF8B5441B7B45805887F051D39BF133B583BAF6860"
+groups="prq,lmn"
+[user2]
+password="BA7816BF8F01CFEA414140DE5DAE2223B00361A396177A9CB410FF61F20015AD"
+groups="xyz"
+[user3]
+password="FADFEF49B40BF551A279F820BD863AC96AEBCBF39B4431DFF4F0D5CB62DD5303"
+
+# scopes
+[scope1]
+groups="xyz,pqr"
+[scope2]
+groups="lmn"
```

Comment:
Does this work with the new Config API change which expects the file to be in TOML?

Reply:
Yes, we have used this in other tests as well.